src/modules/auth/index.js

```diff
@@ -54,12 +56,26 @@ export default class SeaCatAuthModule extends Module {
 			component: ResourcesCreateContainer,
 			resource: "seacat:resource:access"
 		});
+		app.Router.addRoute({
+			path: '/auth/resources-deleted',
+			exact: true,
+			name: 'Deleted resources',
+			component: ResourcesDeletedListContainer,
+			resource: 'seacat:resource:access'
+		});
+		app.Router.addRoute({
+			path: '/auth/resources-deleted/:resource_id',
+			exact: true,
+			name: 'Deleted resource detail',
+			component: DeletedResourceDetailContainer,
+			resource: 'authz:superuser'
+		});
 		app.Router.addRoute({
 			path: '/auth/resources/:resource_id',
 			exact: true,
 			name: 'Resource detail',
 			component: ResourcesDetailContainer,
-			resource: "seacat:resource:access"
+			resource: "authz:superuser"
```

Comment:
@petrKavulok Oh, this resource should have stayed as it was, please put it back

Reply:
@Pe5h4 oh, right, sorry, my mistake, I misunderstood :)